app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt

```diff
@@ -316,9 +317,10 @@ fun questTypeRegistry(
 
     /* ↓ 2.solvable when right in front of it but takes longer to input --------------------- */
 
-    // bike parking: would be higher up if not for bike parking capacity which is usually not solvable when moving past
+    // bike parking/rental: would be higher up if not for bike parking/rental capacity which is usually not solvable when moving past
     AddBikeParkingCover(), // used by OsmAnd in the object description
     AddBikeParkingType(), // used by OsmAnd
+    AddBikeRentalType(),
```

Comment:
Minor re-ordering of the simpler quest:
```suggestion
    AddBikeRentalType(), // generally less overlap of possible types/fewer choices/simpler to answer
    AddBikeParkingType(), // used by OsmAnd
```

Only downside would be if there is rental next to parking it would flip-flop between the two.